dbms/src/Common/OpenSSLHelpers.cpp

```diff
@@ -4,6 +4,8 @@
 #include <ext/scope_guard.h>
 #include <openssl/err.h>
 
+#pragma GCC diagnostic warning "-Wold-style-cast"
```

Comment:
BTW, we don't want any warning in successfull build (we use `-Werror`) because warnings will steal our attention every time.

Reply:
I guess copy/paste "bug" of course it is "ignore"